modules/c++/sys/source/ExecUnix.cpp

```diff
@@ -24,9 +24,120 @@
 #if !defined(WIN32)
 
 #include <sys/wait.h>
-#include "sys/Exec.h"
+#include <str/Manip.h>
+#include <sys/Exec.h>
 
-int sys::ExecPipe::closePipe()
+namespace 
+{
+const size_t readPipe = 0;
```

Comment:
Make these `static` too.  And since they're constants, it should be `READ_PIPE`.  So, `static const size_t READ_PIPE = 0;`